### PR TITLE
[web_embedding] Update README to link to our first community contribs.

### PR DESCRIPTION
**Pull request by ditman**
Update README.md of the `web_embedding` sample to include links to community contributions.

* Follow up of: https://github.com/flutter/samples/pull/1945

## Pre-launch Checklist

- [x] I read the [Flutter Style Guide] _recently_, and have followed its advice.
- [x] I signed the [CLA].
- [x] I read the [Contributors Guide].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing ~~and new~~ tests are passing.

If you need help, consider asking for advice on the #hackers-devrel channel on [Discord].

[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[CLA]: https://cla.developers.google.com/
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[Contributors Guide]: https://github.com/flutter/samples/blob/main/CONTRIBUTING.md

### CODE REVIEW

**Comment by ditman** [comment]
(Of course feel free to change anything on the style/messaging/whatever, this is just a draft!)

**Comment by domesticmouse** [comment]
Tidied up the markdown URLs a bit, other than that, LGTM

**Review by domesticmouse — APPROVED** [review]

File: web_embedding/README.md [hunk 1]
```diff
@@ -9,3 +9,25 @@ This directory contains examples of how to embed Flutter in web apps (without if
...
+
+## Community Contributions
+
+Members of the community have graciously ported and contributed the following examples
+of Flutter Web embedding into other web frameworks:
+
+| Author | Host Framework | Code |
+|--------|----------------|------|
+| [@p-mazhnik][] | **React JS** | [p-mazhnik/flutter-embedding cra-flutter][] |
+| [@p-mazhnik][] | **React Native** | [p-mazhnik/flutter-embedding expo-flutter][] |
```
Comment (p-mazhnik) [inline web_embedding/README.md:21]: I wonder if this sample could be mentioned in the `add_to_app` directory as well. It is an example of embedding Flutter into another cross-platform framework, React-Native (Android, iOS, web). So it demonstrates the possibility of gradual migration from this framework to Flutter, which can be valuable for React-Native developers. 

What are your thoughts on this, @domesticmouse ?

**Review by p-mazhnik — COMMENTED** [review]
Thank you @ditman!

File: web_embedding/README.md (hunk 1 above)
Comment (domesticmouse) [reply web_embedding/README.md:21]: @RedBrogdon we need to think about how to merge our Add to App story with this Flutter for web capability.

**Review by ditman — COMMENTED** [review]
Ship it @domesticmouse! :)

**Pull request merged**